### DISCUSSION TITLE
Error: 500: failed to create user account

### DISCUSSION THREAD

**Discussion by ajm339** [AntlerVC/firetable#916] [Support / Q&A] [answered]
I clicked "Login with Google" which I have done before and got a 500 error on creating an account.

I had to delete Rowy service accounts in GCloud because they were interfering with my FlutterFlow setup - I was logging into Rowy to disable any other settings in there.

**Comment by harinij** [comment, marked as answer]
Hi @ajm339 it would be useful if you can share what the interference in your setup was with FF? Also, if you deleted the access on gcp then that could be the reason for connectivity issue. Are you trying to recreate an account for another project? If you can email us at support@rowy.io we can hop on a quick call to get the issue resolved for you.
